### PR TITLE
configmap parse int

### PR DESCRIPTION
**Pull request by runzexia**
Signed-off-by: RyderXia <ryder.xia@sap.com>
to replace https://github.com/knative/eventing/blob/master/pkg/configmap/reader.go#L36

### CODE REVIEW

**Comment by vagababov** [comment]
it doesn't look 1 to 1 with the function you say to replace?

**Comment by runzexia** [comment]
> it doesn't look 1 to 1 with the function you say to replace?

`ReadInt` is used to read confingmap at https://github.com/knative/eventing/blob/master/pkg/channel/config.go#L54, we can use configmap parse instead, and remove `ReadInt`

**Comment by vagababov** [comment]
I am not against the method itself, but the ReadInt has a list and stuff...

**Comment by runzexia** [comment]
I would create PR in event to discuss readint. My idea is to use utils in 'pkg'  to parse configmap. 😊

File: configmap/parse.go [hunk 1]
```diff
@@ -81,6 +81,20 @@ func AsInt64(key string, target *int64) ParseFunc {
 	}
 }
 
+// AsInt parses the value at key as an uint32 into the target. if it exists.
```
Comment (markusthoemmes) [inline configmap/parse.go:84]:
```suggestion
// AsInt parses the value at key as an into into the target, if it exists.
```

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve
/ok-to-test

**Pull request merged**